save-backend/src/main/kotlin/org/cqfn/save/backend/service/TestSuitesService.kt

```diff
@@ -57,7 +57,7 @@ class TestSuitesService {
      * @return all standard test suites with specific name
      */
     fun findStandardTestSuitesByName(name: String) =
-            testSuiteRepository.findAllByNameAndType(name, TestSuiteType.STANDARD).map { it.toDto() }
+            testSuiteRepository.findAllByNameAndType(name, TestSuiteType.STANDARD)
```

Comment:
Why did you change return type?

Reply:
For now I use this method also from preprocessor, but `dto` is not enough for him, since we need the `id`, which located only in `TesSuite`. So I change the return type and what about `orchestrator` - he called `it.toDto()` by itself